lib/grid.rb

```diff
@@ -13,12 +13,32 @@ def template
     end
 
   def coordinate_valid?(coordinate)
-    coordinates.include?(coordinate)
+    coordinate = coordinate.to_sym
+    coordinates.has_key?(coordinate)
   end
 
-  private
+  def mark(coordinate)
+    coordinate = coordinate.to_sym
+    coordinates[coordinate] = 'X'
```

Comment:
This isn't setting the value like I expect it to. This works in the console. What am I missing here?